charts/minecraft-proxy/templates/deployment.yaml

```diff
@@ -97,7 +97,7 @@ spec:
         {{- end }}
         {{- if .Values.plugins }}
         - name: PLUGINS
-          value: {{ include "helm-toolkit.utils.joinListWithComma" .Values.plugins }}
+          value: {{ join "," .Values.plugins }}
```

Comment:
This path seems to be incorrect, because in [values.yaml](https://github.com/itzg/minecraft-server-charts/blob/2fea96dd82706f6b083b30f5dd9848b10d8b43c6/charts/minecraft-proxy/values.yaml#L85) it's written as `.Values.minecraftProxy.plugins`